### ISSUE TITLE
Elemental Resistances

### ISSUE THREAD

**Issue by DarkHayZe** [Sklore/HL_DD_5e_Colab#617]
I am not sure if this belongs with the community or to LW themselves. However, I just added Earth Elemental to my Druid via the "Shadows of Endore: Adjustments" pack, and I noticed that the resistances and immunities and such are not listed on the stat block. This seems to be true for the other elementals as well. 

They are however listed if you go into the encounter builder, so ...

**Comment by Sklore** [comment]
Sorry I have no idea what the Shadows of Endore: Adjustments pack is. It's not in the community pack to my knowledge. Are you using D&D5e?

**Comment by DarkHayZe** [comment]
This...

[https://github.com/Sklore/HL_DD_5e_Colab/issues/494](Shadows of Endore)

**Comment by Sklore** [comment]
Oh that's not actually been added to the community pack yet. That support should really be provided by LWD as part of the core pack.

**Comment by LordMorin** [comment]
Shadows of Endore is what I call my stuff and is the name of my longstanding campaign. I once asked ShadowChemosh if he'd add my personal Adjustment and stuff under that soun=\rce but I've never seen anything added under it but my own stuff.

The only thing I've created for druids is an Adjustment for druids that is meant to be added to druid hirelings to change them into druid forms. 

I'm looking into this as I type.

**Comment by LordMorin** [comment]
This is mine. This was a work in progress that never got added to the community pack to the best of my knowledge. I never tested it with elemental forms.

As sklore said when I posted it and suspended the post and file, it'sDruid forms is best something handled by LW but to date nothing has yet been done about it from the LW end. 

I'd actually forgotten I'd put it up on GIT for peeps to use make use of if they wanted it.

I still use it myself, and I think I tweaked it some, but it has been a while since I last looked at it.

I'll take another look at it and see what I can do.

My druid never made it to a high enough level for Elelemtal forms.

**Comment by DarkHayZe** [comment]
I don't believe it is an issue with the Druid pack itself, because if I go to select an elemental as a race the resistances are not there either. I just wasn't sure if the monster manual fell within the purview of the community pack or not.

**Comment by Sklore** [comment]
Anything in the SRD is handled by LWD. Anything else is handled by us.

**Comment by DarkHayZe** [comment]
Fair enough, like I said I wasn't sure. Didn't realize the MM was SRD

**Comment by Sklore** [comment]
@LordMorin  that explains it. My Googling was only bringing up Star Wars. 
@DarkHayZe all good mate, the editor side of the tool can be quite a big tool to get your heard around and most people struggle to determine which items are supported by us or not. A key identifier is anything with 5C in the unique name is created and supported by us.  You seem to have already come leaps and bounds beyond 95% of the users. Happy to bring you onto the team if you would like to contribute directly to the project. 

FYI guys. I'm doing some other stuff for LWD on the side which is why I've been so quiet lately. There was a lull this last week which is why I popped my head back up back that all changed over-night so I'm going back into my cave :) Scream if you need me, I am watching.

**Comment by DarkHayZe** [comment]
I won't say no...lol

My weakness is scripting, I could stand to learn a lot more in that department.

**Comment by Sklore** [comment]
You and me both. I'm a shitty scripter. I've got the wiki going here with common script that we use. Might help.

**Comment by DarkHayZe** [comment]
I looked at it, and while I understand some basics the language being used is far from ideal.

I do have a couple updates that will fix some issues I have stumbled across in the community pack, mostly while building my custom class/archetype. And I just have one issue left on that before I consider it complete, and one enhancement but from the look of it, it might not be possible for the enhancement... shrug.
